lib/template.html

```diff
@@ -104,6 +104,10 @@
   <meta property="og:title" content="{{ seo_page_title }}" />
 {% endif %}
 
+{% if seo_author %}
+  <meta name="author" content="{{ seo_author }}" />
+{% endif %}
+
```

Comment:
Added in content of PR #103 along with tests (because it made sense to add the author meta field when I was already doing it in the JSON).